grails-app/views/report/_report.gson

```diff
@@ -1,22 +0,0 @@
-import streama.Episode
```

Comment:
those are the wrong gsons to remove, i think!

Reply:
I think they weren't used anymore after this. That one change where I changed from gsons to json.use a bit above was the only case I used these gsons, so I removed them after changing to json.use

Reply:
But the ones under /video/_report.gson were still present